.github/workflows/sqlx.yml

```diff
@@ -314,6 +314,17 @@ jobs:
           # but `PgLTree` should just fall back to text format
           RUSTFLAGS: --cfg postgres_${{ matrix.postgres }}_client_ssl
 
+      - if: matrix.tls != 'none' && matrix.postgres == '17'
+        run: >
+          cargo test
+          --no-default-features
+          --features any,postgres,macros,_unstable-all-types,runtime-${{ matrix.runtime }},tls-${{ matrix.tls }}
+        env:
+          DATABASE_URL: postgres://postgres@localhost:5432/sqlx?sslnegotiation=direct&sslmode=require&sslkey=.%2Ftests%2Fkeys%2Fclient.key&sslcert=.%2Ftests%2Fcerts%2Fclient.crt
+          # FIXME: needed to disable `ltree` tests in Postgres 9.6
+          # but `PgLTree` should just fall back to text format
+          RUSTFLAGS: --cfg postgres_${{ matrix.postgres }}_client_ssl
+
```

Comment:
Now that https://github.com/launchbadge/sqlx/pull/3950 is merged, this additional step in the job should be moved to the new job which only runs TLS tests. Note that the `matrix.tls != none` is no longer needed (since that job is only for tls), but the postgres version check still is.

The fixme can also be deleted here like it was in that pr, and the `--cfg` passed to `RUSTFLAGS` should be similarly adjusted.

Reference:
https://github.com/launchbadge/sqlx/pull/3950/files#diff-73f3a0cbafbafd570d1acd781a2fb75877e89a04d49ab813c3a22c152bc4e470